### ISSUE TITLE
"Queue" network [undefined]

### ISSUE THREAD

**Issue by cristiandley** [arunoda/meteor-up-legacy#150]
Really dont know if this issue is the right repository.

But if run "meteor" in my local machine i got no problem.

When i deploy i get "net::ERR_CONNECTION_REFUSED" and "Queue" network [undefined]"

``` bash
GET http://mydomain.com/sockjs/info?cb=7sbwqvwd9v net::ERR_CONNECTION_REFUSED 
```

(sockjs ?)

and this issue if i try to upload an image:
- i already installed power-quere or micro-queue

``` bash
Error: "Queue" network [undefined], Error
    at http://mydomain.com/d4113048f6873f06ed8b5ce655e8b1f5a39feb00.js:22:23751
    at http://mydomain.com/d4113048f6873f06ed8b5ce655e8b1f5a39feb00.js:22:21121
    at http://mydomain.com/d4113048f6873f06ed8b5ce655e8b1f5a39feb00.js:1:8368
    at XMLHttpRequest.v.onreadystatechange (http://mydomain.com/d4113048f6873f06ed8b5ce655e8b1f5a39feb00.js:22:22428) 
```

this issue is here also, [here](https://github.com/CollectionFS/Meteor-CollectionFS/issues/217)
but for me is only happening when i run it under meteor-up

**Comment by cristiandley** [comment]
``` bash
meteor add cfs:gridfs
```

is the solution to people using, standart pacakge dont install gridfs

``` bash
cfs:access-point              0.0.0*                                                                  
cfs:collection                   0.0.1                                                                         
cfs:filesystem                  0.0.0
cfs:http-methods            0.0.24                                                                             
cfs:power-queue            0.0.1 
```

still dont understund why on my local machine works...

**Comment by DanielDornhardt** [comment]
Just had the same issue, I had to meteor add cfs:gridfs on my server too, even though I use the filesystem storage, after that both display and upload of images work again using meteor-up.

**Comment by jsam** [comment]
Adding cfs:gridfs does not work for me.

I still have this error when uploading through HTTPS, error is not present when uploading on HTTP, only this warning:

  The provided value 'undefined' is not a valid enum value of interface XMLHttpRequestResponseType.

**Comment by mattiLeBlanc** [comment]
I have the same Issue. 
At first I used cfs:filesystem, then I ran into some path issues etc....and I decided to add CFS:Gridfs. That worked but then I decided to go back to cfs:filesystem.
Which worked after I fixed the path issue.

Then I removed CFS:GridFS and its mongo directories and kept on developing. Nothing wrong locally, but when I did a mup deploy and wanted to upload a file I suddenly get the
`"Queue" network [undefined], Error: network [undefined]` error.

Now I have added the CFS:gridfs package back in and upload via CFS:filesystem works again.
very strange.
